README.md

```diff
@@ -1,13 +1,32 @@
 # Uppy.io
 
-Website and documentation for [uppy](https://uppy.io/).
+Website and documentation for [uppy.io](https://uppy.io).
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern
-static website generator.
+Code for [uppy.io/blog](https://uppy.io/blog) lives in this repo.  
+Code for [uppy.io/docs](https://uppy.io/docs/quick-start) lives in
+[github.com/transloadit/uppy](https://github.com/transloadit/uppy).
```

Comment:
```suggestion
[transloadit/uppy](https://github.com/transloadit/uppy).
```

Reply:
Why? I feel `github.com` prefix makes it more intelligible, and removes the need to remember github-specific shortcuts.